src/%IPM/Lifecycle/DeployedModule.cls

```diff
@@ -1,8 +0,0 @@
-/// Module lifecycle, but with different default handling for deployment of resources. If no value is specified for the "Deploy" attribute,
-/// the resource is marked for deployment.
-Class %IPM.Lifecycle.DeployedModule Extends %IPM.Lifecycle.Module
```

Comment:
Should check if any of our modules are using this. I don't see any from this p4 grep command though so should be safe:
```
p4 grep -n -s -e "DeployedModule" //healthshare/appmodules/.../module.xml
```